### ISSUE TITLE
using latest serenity-cli

### ISSUE THREAD

**Issue by tremkonzah** [jan-molak/serenity-js#4]
How to use latest version of serenity-cli jar when running `node_modules/serenity-cli/bin/serenity update`

its always download the old version

trying to modify config.js to 
```
artifact: 'net.serenity-bdd:serenity-cli:jar:1.1.37-rc.6',
    repository: 'https://dl.bintray.com/serenity/maven/'
```

running `npm run report` produce an error
```
error: no main manifest attribute,
(node:14860) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 3): Error: no main manifest attribute,
```

**Comment by jan-molak** [comment]
Serenity-cli jar follows a different versioning strategy than the main Serenity BDD library. The latest version available ([0.0.1-rc.3](https://bintray.com/serenity/maven/serenity-cli)) is the one that's specified in the Serenity CLI node module.

Does that answer your question?

**Comment by tremkonzah** [comment]
thanks for your clarification
